runtime/onert/core/src/backend/builtin/train/KernelGenerator.cc

```diff
@@ -67,12 +67,9 @@ void KernelGenerator::visit(const ir::train::operation::Permute &node)
   output_back_prop_tensors.emplace_back(output_back_prop_tensor);
   input_back_prop_tensors.emplace_back(input_back_prop_tensor);
 
-  // NOTE IOTensors of graph outputs for passing data to users must be ignored in training
-  //      because the buffers of those IOTensors are unnecessary and nullptr
-  bool ignore_forward_in_training = _whole_graph_outputs.contains(output_index);
```

Comment:
I have a question.
Currently, Permute layer is inserted and works when trying to train a model?

Reply:
> Currently, Permute layer is inserted and works when trying to train a model?

Yes, right. To calculate the accuracy in training mode, the output data is required.

Reply:
I see. I guess inserting Permute layer is involved with `IOTensor`s now. But, I think it would be better if Permute layer is not inserted for `IOTensor`s ultimately. I will look at it later.
Anyway, there is no problem if this code is removed.